test/e2e/components/vl-properties.js

```diff
@@ -1,10 +1,61 @@
 const { VlElement } = require('vl-ui-core').Test;
+const { By } = require('vl-ui-core').Test.Setup;
 
-class VlProperties extends VlElement {  
-    constructor(driver, selector) {
-        super(driver, selector);
+class VlProperties extends VlElement {
+    async getSlotElements() {
+        const slot = await this.shadowRoot.findElement(By.css('slot'));
+        return this.getAssignedElements(slot);
     }
+}
+
+class VlPropertiesColumn extends VlElement {
+    async isFullSize() {
+        return this.hasAttribute('full');
+    }
+
+    async getPropertiesList() {
+        return new VlPropertiesList(this.driver, await this.findElement(By.css('[is="vl-properties-list"]')));
+    }
+}
+
+class VlPropertiesList extends VlElement {
+    async getProperty(term) {
+        const properties = await this.getProperties();
+        for (let i = 0; i < properties.length; i++) {
+            const property = properties[i];
+            if (property.term && await property.term.getText() === term) {
+                return property;
+            }
+        }
+    }
+
+    async getProperties() {
+        const terms = await this._getTerms();
+        const values = await this._getValues();
+        return terms.map((term, index) => {
+            const value = values[index];
+            return {
+                term: term,
+                value: value
+            }
+        });
+    }
+
+    async _getTerms() {
+        const rawTerms = await this.findElements(By.css('[is="vl-property-term"]'));
+        return Promise.all(rawTerms.map(term => new VlPropertyTerm(this.driver, term)));
+    }
+
+    async _getValues() {
+        const rawValues = await this.findElements(By.css('[is="vl-property-value"]'));
+        return Promise.all(rawValues.map(value => new VlPropertyValue(this.driver, value)));
+    }
+}
+
+class VlPropertyTerm extends VlElement {
+}
 
+class VlPropertyValue extends VlElement {
 }
```

Comment:
dit draagt (voorlopig) niet heel veel bij ...

Reply:
Nee, dat klopt. Maar we doen dat op ander plaatsen ook. Vorige keer dat ik dat besproken had met Tom was de conclusie dat we beter wel al een klasse hebben hiervoor ook al is die leeg. @coemans , jij nog input?